### ISSUE TITLE
CTC loss error

### ISSUE THREAD

**Issue by ghost** [novioleo/FOTS#14]
RuntimeError:

**Comment by novioleo** [comment]
@jahongir7174 target doesn't convert to cuda.

**Comment by ghost** [comment]
Thank you, I have solved the problem.
it works for me ;)
"loss = self.ctc_loss(pred[0].to(device), gt[0].to(device), pred[1].to(device), gt[1].to(device))"
